### ISSUE TITLE
Tokenization should be optional

### ISSUE THREAD

**Issue by mrsweaters** [refile/refile#441]
Having the token in the attachment_url should be enabled by default, but can be disabled to be backwards compatible with 0.5.x. I'm currently stuck on 0.5.x and see no easy way of migrating to the newer release. The problem is painful if you have a highly distributed application environment that requires caching these asset urls.

https://github.com/refile/refile/issues/388#issuecomment-182603558

**Comment by Padchi** [comment]
+1 same here. I have stored ~4 (depends on the image dimensions) URLs with different sizes of an image in the database and now I have no idea how I should upgrade this. Storing urls in database may not be the best thing but it's a API only application and the client expects an array of images for each attachment. Similar to what Spotifys api returns for a image

**Comment by mrsweaters** [comment]
@jnicklas Should I submit a pull request to resolve this issue?

**Comment by mrsweaters** [comment]
@jnicklas Any update on this would be awesome! :)

**Comment by mrede** [comment]
+1 to this too. I've just updated from 0.5x to 0.6.x and had some head scratching moments while my React front end failed to load any images. We had created a way for the front end to generate the Refile image sizes on the fly and now this doesn't work.

**Comment by mrsweaters** [comment]
@mrede We did the same thing. Our application is essentially multiple rails apps that are coupled together with a gem we have. This gem provides helpers to generate links to common image dimensions across apps. We also have to cache these urls in some of the apps which makes upgrading very difficult. 

It does seem that I could copy over the logic from the tokenization method and get it to work, but I would like to avoid this if possible.

**Comment by Padchi** [comment]
I ended up with create the urls on the fly from my API and served them to my ember app. Dunno if this the best option cuz I haven't pushed this to production yet but when I do we need to recatch everything and it's gonna be a pain in the ass.
Last time I recatched everything our site crashed for 24 hours and I guess I need to to do a cronjob or something before I push everything to production.

Seems like this gem is dead and that's the fucking problem, we are stuck whit this.. 

Skickat från min iPhone
